### ISSUE TITLE
Support Tiled maps

### ISSUE THREAD

**Issue by scottbw** [RSamaium/RPG-JS#47]
It would be great to be able to edit maps using Tiled

**Comment by neil-119** [comment]
@jrconway3 Do you mean that RPG-JS supports Tiled natively? How did you get it to work?

**Comment by jrconway3** [comment]
They added support for Tiled natively now.


Additionally, BEFORE they added this support, I had gotten it to work, but I had to make some modifications to the core RPG-JS code to add additional plugin regions so I could get it to work. I never asked to merge request this because it wasn't ready yet.

I also created a Tiled Passability Map script that seemed to work decently well. Now with this latest change it won't work; I still have to go back in and make it work with their Tiled support.
